invenio_circulation/config.py

```diff
@@ -102,6 +106,11 @@
         duration_validate=is_loan_duration_valid,
         item_available=is_item_available
     ),
+    extension=dict(
+        from_end_date=extend_from_end_date,
```

Comment:
`duration_default` and `max_count` can be changed by the ILS, but `from_en_date` is just a boolean. Thus we can replace `extend_from_end_date` can be removed by a boolean value.